CHANGES.md

```diff
@@ -1,5 +1,11 @@
 ## [Friend](http://github.com/cemerick/friend) changelog
 
+### `0.3.x`
```

Comment:
Not sure how does versioning works these days - should I replace `x` with something specific?

Reply:
No worries, I'll take care of that at release